testing/helpers/checkStyleHelper.js

```diff
@@ -0,0 +1,53 @@
+import devices from "core/devices";
+import browser from "core/utils/browser";
+
+function testInChromeOnDesktop(name, testCallback) {
+    if(devices.real().deviceType === "desktop" && browser.webkit) {
+        QUnit.testInActiveWindow.call(null, name, testCallback);
+    } else {
+        QUnit.skip.call(null, name, testCallback);
+    }
+}
+
+function checkTextOverflow(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.textOverflow, expectedValue, "textOverflow");
+}
+
+function checkWhiteSpace(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element);
+    assert.equal(style.whiteSpace, expectedValue, "whiteSpace");
+}
+
+function checkOverflowX(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.overflowX, expectedValue, "overflowX");
```

Comment:
I believe we should use `strictEqual` anywhere, even if it's not critical here

Reply:
Yes, I used equal instead of strictEqual by a mistake. I will change it.

Reply:
Done.